### PR TITLE
PAYARA-2171 Add timeout parameter to start instance command

### PR DESCRIPTION
**Pull request by MattGill98**

### CODE REVIEW

**Comment by MattGill98** [comment]
jenkins test please

File: nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/StartInstanceCommand.java [hunk 1]
```diff
@@ -277,21 +294,29 @@ private void startInstance(AdminCommandContext ctx) {
...
             }
+        }, 500, TimeUnit.MILLISECONDS);
+
+        // If the timeout is reached, return the timeout message. Otherwise return null (success).
+        try {
+			instanceTimeout.await(timeout, TimeUnit.SECONDS);
+		} catch (InterruptedException e) {
+			return Strings.get("start.instance.timeout", instanceName);
+		} finally {
+            executor.shutdown();
```
Comment (arjantijms) [inline nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/StartInstanceCommand.java]: Maybe put the "try" higher up? If `executor.schedule` throws an exception, the executor might not be shut down.

**Comment by MattGill98** [comment]
jenkins test please

File: nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/StartInstanceCommand.java (hunk 1 above)
Comment (MattGill98) [reply nucleus/cluster/admin/src/main/java/com/sun/enterprise/v3/admin/cluster/StartInstanceCommand.java]: Good point, I've moved it higher up

**Pull request merged**